base/shell/cmd/lang/ja-JP.rc

```diff
@@ -180,7 +180,7 @@ DIR [ドライブ:][パス][ファイル名] [/A[[:]属性]] [/B] [/C] [/D] [/L]
     STRING_DIR_HELP5 "\n     ファイルの総数:\n%16i 個のファイル% 14s バイト\n"
     STRING_DIR_HELP6 "%16i 個のディレクトリ% 15s バイト\n"
     STRING_DIR_HELP7 "\n %s のディレクトリ\n\n"
-    STRING_DIR_HELP8 "%16i 個のファイル% 14s バイト\n"
+    STRING_DIR_HELP8 "%16i 個のファイル    % 15s バイト\n"
```

Comment:
I don't know whether we have korean translation in cmd yet, but if we do, should a similar change be done also in korean? (if not, is it because the formatter turned out to be already correct there?)

Reply:
Korean also uses double-width characters.
HELP8 should be aligned as HELP6 does.

Reply:
Here `バイト` is for bytes.
Aligning units is a natural format.

Reply:
So in the korean translation is already OK regarding the formatter? or does it need to be fixed? :sweat_smile:

Reply:
I cannot browse. I have no PC now.

Reply:
There is no `ko-KR.rc`.